x-pack/plugins/security_solution/server/fleet_integration/fleet_integration.ts

```diff
@@ -116,6 +121,8 @@ export const getPackagePolicyUpdateCallback = (
       return newPackagePolicy;
     }
 
+    notifyProtectionFeatureUsage(newPackagePolicy, featureUsageService, endpointMetadataService);
```

Comment:
Should this check be done before the validation agains the policy below?

If wanting to track usage, it might be best to do it after validation, since that process (I think) adjust the policy if it finds features that are not allowed under the current policy.